src/dreaditor/painters/worldgraph.py

```diff
@@ -1,24 +1,22 @@
 import math
 
-from PySide6.QtGui import QPainter, QColor, QPen
+from PySide6.QtGui import QFont, QPainter, QColor, QPen
 from PySide6.QtWidgets import QStyleOptionGraphicsItem, QWidget
 from PySide6.QtCore import QRectF, QPointF
 
 from dreaditor.actor import Actor
 
-
-COLOR = QColor(255, 255, 255, 128)
+GRAPH_COLOR = QColor(255, 255, 255, 128)
+TEXT_COLOR = QColor(0, 0, 0, 150)
```

Comment:
I might bump this to 255 alpha since it's text, though it depends on how readable the text is.